packages/ripple-nuxt-tide/modules/site/module.js

```diff
@@ -3,7 +3,7 @@ const path = require('path')
 module.exports = function () {
   const options = this.options.tide
   const { URL } = require('url')
-  const baseUrl = new URL(options.baseUrl)
+  const baseUrl = options.baseUrl ? new URL(options.baseUrl) : ''
```

Comment:
I added extra validation as Lagoon build failed throwing FATAL error on `URL()` with invalid value in options.baseUrl.

Reply:
I think we should throw an error and stop the build if `baseUrl` is not set, because without `baseUrl`, none of the Tide integration will work. We just need log a meaningful error message.  Maybe we should do validation for mandatory config option at the beginning of the build [(here)](https://github.com/dpc-sdp/ripple/blob/master/packages/ripple-nuxt-tide/lib/module.js#L8) How do you think?

Reply:
Thanks for the suggestion @tim-yao. I'll look into this.